### DISCUSSION TITLE
How do you edit the PR commit message?

### DISCUSSION THREAD

**Discussion by megasanjay** [semantic-release/semantic-release#2514] [Configuration] [answered]
We use the semantic releases via GitHub actions and we use a secondary account as the GitHub token. This works as intended but we would like to slightly edit this message to be more informative. Is there a config option we can use to edit this message?

![image](https://user-images.githubusercontent.com/21206996/180871719-1c63f3db-3b78-49b6-841b-e4977dd4fcc9.png)

**Comment by travi** [comment, marked as answer]
https://github.com/semantic-release/github#options
